Fix/uppsf 1891 spark del secret

epavlova opened the pull request:
# Description

## What

Add basic auth credentials for delivery staging and production environments. The new credentials will be shared with Spark team. The LastPass notes are already updated.

## Why

https://financialtimes.atlassian.net/browse/UPPSF-1891


## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [X] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

bboykov reviewed (dismissed):
Please share the name of the user in the comments. I suppose this is also updated in LP.

epavlova commented:
> Please share the name of the user in the comments. I suppose this is also updated in LP.

The name of the new user is `spark-prod-20201112` for production and `spark-pre-prod-20201112` for staging environment. Yes, it's already in LastPass but it's not on the clusters, I will deploy it with this PR.

Btw I noticed that LastPass is missing some credentials for staging environment and I added them in the LastPass note.
